### PR TITLE
Login menu scaling

### PR DESCRIPTION
**Pull request by Sammy-T**
This attempts to address the slight cutoff that occurs on smaller screens.
Here is how the login page appears on my phone screen:
 

I've edited the styles to wrap the input fields and maintain a little padding from the screen edge:

### CODE REVIEW

File: components/layout/FormLayout.module.css [hunk 1]
```diff
@@ -11,14 +11,22 @@
 
 .row {
   display: flex;
+  flex-wrap: wrap;
   align-items: center;
   position: relative;
   margin-bottom: 20px;
   line-height: 1.8;
+  padding: 0 6px;
```
Comment (gnarlex) [inline components/layout/FormLayout.module.css:19]: This is just personal preference but I'd increase the padding to something around `20px`

Before - After:
Screenshot 2021-03-08 at 09 31 51 Screenshot 2021-03-08 at 09 23 35

**Review by gnarlex — APPROVED** [review]
LGTM 👍   Thanks for fixing the login screen for small phones

File: components/layout/FormLayout.module.css (hunk 1 above)
Comment (Sammy-T) [reply components/layout/FormLayout.module.css:19]: Thanks. Yeah, that would be fine with me. My concern was just to have it fit on the screen. Before changing it though, I'd like to know if @mikecao also agrees with increasing the padding or has any additional concerns.

Comment (mikecao) [reply components/layout/FormLayout.module.css:19]: Yea, a little more padding would be nice.

Comment (Sammy-T) [reply components/layout/FormLayout.module.css]: No problem. I've adjusted it.

**Pull request merged**